tools/check-source.sh

```diff
@@ -108,9 +108,9 @@ grep -Hne '^\\\(change\|rationale\|effect\|difficulty\|howwide\)\s.\+$' compatib
     fail "change marker in [diff] followed by stuff" || failed=1
 # Fixup: sed 's/^\\\(change\|rationale\|effect\|difficulty\|howwide\)\s\(.\)/\\\1\n\2/'q
 
-# "template <class" (with space) in library clause.
-grep -ne 'template\s<class' $texlib |
-    fail 'space between "template" and "<class"' || failed=1
+# "template <" (with space) in library clause.
+grep -ne 'template\s<' $texlib |
```

Comment:
We should probably catch any number of whitespace characters instead of exactly one:
```suggestion
grep -ne 'template\s+<' $texlib |
```

Reply:
That should be `\+` because this is a POSIX BRE not ERE. Use `grep -E` for EREs.

Reply:
Good point. Fixed.